app/controllers/navigation/Navigator.scala

```diff
@@ -45,3 +49,28 @@ class Navigator @Inject()(appConfig: AppConfig, auditService: AuditService) {
   }
 
 }
+
+object Navigator {
+
+  val standard: PartialFunction[DeclarationFieldCompanion, Mode => Call] = {
+    case BorderTransport => controllers.declaration.routes.DepartureTransportController.displayPage
+    case _               => ???
+  }
+
+  val supplementary: PartialFunction[DeclarationFieldCompanion, Mode => Call] = {
+    case BorderTransport => controllers.declaration.routes.DepartureTransportController.displayPage
+    case _               => ???
+  }
+
+  val simplified: PartialFunction[DeclarationFieldCompanion, Mode => Call] = {
+    case BorderTransport => controllers.declaration.routes.WarehouseIdentificationController.displayPage
+    case _               => ???
+  }
+
+  def backLink(page: DeclarationFieldCompanion)(implicit request: JourneyRequest[_]): Mode => Call =
```

Comment:
I suspect we will need more than just knowledge of the page to go back, some pages will take into account previous answers etc

Reply:
I would like to do not make upfront work because we really don't know how to change it